binding.gyp

```diff
@@ -1,150 +1,185 @@
 {
-  "targets": [
-    {
-      "target_name": "pact",
-      "sources": [
-        "native/addon.cc",
-        "native/ffi.cc",
-        "native/consumer.cc",
-        "native/provider.cc",
-        "native/plugin.cc"
-      ],
-      "include_dirs": [
-        "<!(node -p \"require('node-addon-api').include_dir\")",
-        "<(module_root_dir)/native",
-        "<(module_root_dir)/ffi",
-      ],
-      "conditions": [
-        [
-          "OS=='win'",
-          {
-            "libraries": [
-              "<(module_root_dir)/ffi/pact_ffi.dll.lib"
+    "variables": {
+        "is_alpine": "<!(grep -q Alpine /etc/os-release && echo true || echo false)"
```

Comment:
This file is essentially the same despite the white space changes.

We check if the os is alpine and then build the musl bindings.

node-gyp-build detects musl at runtime when loading the libraries, so will correctly pick between musl/glibc as required